From: Petr Uzel <petr.uzel@suse.cz>
Subject: build: Patch fatresize build into parted package
Patch-mainline: no, custom SUSE patch

Fatresize is used by YaST to resize FAT16/FAT32 file systems. It uses
libparted functions for that purpose but does not resize the FAT
partition afterward. That is the task of parted instead. So using it
without parted and YaST doesn't make much sense.

Versions more recent than 0.1 like e.g. 1.0.3 are incompatible and
risk data loss by silently increasing the FS size (See: bsc#1072479).

Upstream: https://github.com/ya-mouse/fatresize
[sparschauer: Wrote the patch description]
---
 Makefile.am  |    2 +-
 configure.ac |    1 +
 2 files changed, 2 insertions(+), 1 deletion(-)

Index: parted-3.1/Makefile.am
===================================================================
--- parted-3.1.orig/Makefile.am
+++ parted-3.1/Makefile.am
@@ -1,5 +1,5 @@
 ACLOCAL_AMFLAGS = -I m4
-SUBDIRS = po lib include libparted parted partprobe doc debug tests
+SUBDIRS = po lib include libparted parted partprobe fatresize doc debug tests
 
 EXTRA_DIST =				\
   .version				\
Index: parted-3.1/configure.ac
===================================================================
--- parted-3.1.orig/configure.ac
+++ parted-3.1/configure.ac
@@ -610,6 +610,7 @@ libparted/tests/Makefile
 libparted.pc
 parted/Makefile
 partprobe/Makefile
+fatresize/Makefile
 doc/Makefile
 doc/C/Makefile
 doc/pt_BR/Makefile
